sdk/keyvault/azure-security-keyvault-certificates/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "java",
   "TagPrefix": "java/keyvault/azure-security-keyvault-certificates",
-  "Tag": "java/keyvault/azure-security-keyvault-certificates_e018897fab"
+  "Tag": "java/keyvault/azure-security-keyvault-certificates_f79afcd807"
```

Comment:
Why did you need new recordings? Does this PR have new features not already on main?

Reply:
The ordering of a few URL query parameters for paging operations got changed with the newly generated code.